lib/routes/gelbooru/post.ts

```diff
@@ -0,0 +1,113 @@
+import { Route, ViewType } from '@/types';
+import { Context } from 'hono';
+import { parseDate } from '@/utils/parse-date';
+import { renderDesc, getAPIKeys } from './utils';
+
+import got from '@/utils/got';
+import queryString from 'query-string';
+
+export const route: Route = {
+    path: '/post/:tags?/:quality?',
+    categories: ['picture'],
+    view: ViewType.Pictures,
+    example: '/gelbooru/post/1girl rating:general',
+    parameters: {
+        tags: '要搜索的标签，多个标签用 ` `（空格）隔开',
+        quality: {
+            description: '图片质量，可选值为 `sample`（压缩后的图片，推荐值） 或 `orig`（原图），默认为 `sample`',
+            default: 'sample',
+        },
+    },
+    features: {
+        requireConfig: [
+            {
+                name: 'GELBOORU_API_KEY',
+                description: 'Gelbooru 偶尔会开启 API 认证，需配合 `GELBOORU_USER_ID`，从 `https://gelbooru.com/index.php?page=account&s=options` 获取',
+                optional: true,
+            },
+            {
+                name: 'GELBOORU_USER_ID',
+                description: '参见 `GELBOORU_API_KEY`',
+                optional: true,
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportRadar: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['gelbooru.com/index.php'],
+        },
+    ],
+    name: 'Gelbooru标签查询',
```

Comment:
Route name should not include namespace name https://github.com/DIYgod/RSSHub/blob/101637a43737a71b8a17c86cf0ecdb514fd3b172/lib/routes/gelbooru/namespace.ts#L4 again
```suggestion
    name: '标签查询',
```